lib/bots/versionbot.ts

```diff
@@ -354,7 +341,7 @@ export class VersionBot extends ProcBot {
         generateWaffleReference(pr.body);
 
         // Now look through all the commits in the PR. Do the same thing.
-        return this.githubCall({
+        return this.dispatchToEmitter(this.githubEmitterName, {
```

Comment:
Possibly naive question, but rather than change every instance of `this.githubCall(...)` to `this.dispatchToEmitter(this.githubEmitterName, ...)`, would it have made sense to just change the `githubCall` method to something like
```
githubCall(args) {
    return this.dispatchToEmitter(this.githubEmitterName, args);
}
```
? (using whatever the relevant TypeScript syntax is)